src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -65,7 +63,7 @@ def __init__(self, config=None):
         self.register(self.COMMAND_PARSER_PARSED, self._handle_builtin_arguments)
 
         # Let other extensions make their presence known
-        azure.cli.extensions.register_extensions(self)
+        azure.cli.core.extensions.register_extensions(self)
```

Comment:
>register_extensions(self) [](start = 34, length = 25)

Okay, I can see this call being part of the core, but it seems the extensions themselves should not be part of the core... or they aren't really extensions. ;)